bit_manipulation/quine_macluskey.cpp

```diff
@@ -0,0 +1,404 @@
+/**
+ * @file quine_macluskey.cpp
+ * @author [jang-jaehyuk](https://github.com/jang-jaehyuk)
+ * @brief Implementation of [minimization of
+ * Boolean functions](https://en.wikipedia.org/wiki/Quine-McCluskey_algorithm).
+ */
+
+#include <iostream> /// for io operations
+#include <vector> /// for store the implicant
+#include <algorithm> /// for sorting implicant
+
+using namespace std;
```

Comment:
Let use `using std::cout` instead. 🙂
```suggestion
using std::cout;
using std::vector;
using std::cin;
using std::endl;
```

If I missed any, please let me know.